domains/certificates/Query.API/API.IntegrationTests/Repositories/SyncStateTests.cs

```diff
@@ -10,8 +10,6 @@
 using DataContext.ValueObjects;
 using FluentAssertions;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.Logging;
-using NSubstitute;
 using Testing.Helpers;
```

Comment:
Class rename to something with slidingwindow instead of syncstate

Reply:
Done